### PR TITLE
Update breakfast.md

### PR DESCRIPTION
**Pull request by ccastilloux**
## 🎯 Objective

Update breakfast 🍳 with recipe and instructions.

## 🛠️ Changes made

Proposing to make eggs and bacon and included instructions on how to do so.

## 👥 Who needs to review this Pull Request? 
@Justinli90 @lancebarnardberkeley

### CODE REVIEW

**Comment by Justinli90** [comment]
@ccastilloux can we consider thick cut bacon instead of regular bacon? 😋😋

**Comment by lancebarnardberkeley** [comment]
should we do scrambled or poached?? @ccastilloux

**Comment by ccastilloux** [comment]
@Justinli90 No problem 👍🏽
@lancebarnardberkeley Good call, made it so eggs can be made to the cook's preference!

**Comment by Justinli90** [comment]
In my experience it takes closer to 20 minutes to cook a recipe like this. Changing to 20 minutes! Feel free to change back if you disagree 😁

Looks delicious guys!

@ccastilloux @lancebarnardberkeley

**Comment by Aleixsc** [comment]
This looks delicious, but also a bit unhealthy. 
Could we add the option to use egg whites instead of whole eggs?

@ccastilloux  @lancebarnardberkeley @Justinli90

**Comment by Aleixsc** [comment]
Changing the title to "Eggs and Crispy Bacon"

File: breakfast/readme.md [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
 
 
-## Equipment needed:
+## Equipment needed: Stove/cook top, 1 fying/cooking pan, large mixing bowl
 
 
 ## Steps for preparation:
-
-
+1. Crack all 8 eggs into a mixing bowl
```
Comment (Aleixsc) [inline breakfast/readme.md]: Or crack 2 full eggs into a bowl, then separate white from yolks in another bowl, and add the whites to the 2 eggs.

**Review by Aleixsc — CHANGES_REQUESTED** [review]

**Comment by Justinli90** [comment]
> This looks delicious, but also a bit unhealthy.
> Could we add the option to use egg whites instead of whole eggs?
> 
> @ccastilloux @lancebarnardberkeley @Justinli90

@Aleixsc great call. Less cholesterol. Thanks for making these changes!

**Comment by Justinli90** [comment]
@lancebarnardberkeley @ccastilloux @Aleixsc If changes look good to you all, I will merge the PR. Thanks! 👊💪

**Comment by ccastilloux** [comment]
@lancebarnardberkeley @Justinli90 @Aleixsc Looks good to me as well 👍

**Comment by Justinli90** [comment]
Approved from my side as well. Merging, thanks!  👍 😄

**Pull request merged**